tasks/configure.yml

```diff
@@ -18,6 +18,16 @@
       become: true
       become_user: "{{ react_local_user }}"
 
+    - name: Transpile packages
+      command: yarn run lerna:prepublish
```

Comment:
Could we make this command configurable?  On our repo we have this as `yarn run lerna:prepublish` but for others, or even for us on other repos, this command may be different.